metricflow/model/parsing/schemas.py

```diff
@@ -155,7 +155,7 @@
         },
     },
     "additionalProperties": False,
-    "required": ["name", "window_choice"],
+    "required": ["name"],
```

Comment:
For validating aggregation type we could consider making it a restricted enum in the JSON schema here. It does seem cleaner to add a custom enum type for this, though - I suspect we'll need it for stuff like SCDs as well.